tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -2035,6 +2040,52 @@ def test_apdex_transactions(self):
         assert len(response.data["groups"]) == 1
         assert response.data["groups"][0]["totals"] == {"transaction.apdex": 0.6666666666666666}
 
+    def test_miserable_users(self):
+        user_ts = time.time()
+        self._send_buckets(
+            [
+                {
+                    "org_id": self.organization.id,
+                    "project_id": self.project.id,
+                    "metric_id": self.tx_user_metric,
+                    "timestamp": user_ts,
+                    "tags": {
+                        self.tx_satisfaction: indexer.record(
+                            self.organization.id, TransactionSatisfactionTagValue.FRUSTRATED.value
+                        ),
+                    },
+                    "type": "s",
+                    "value": [1],
+                    "retention_days": 90,
+                },
+                {
+                    "org_id": self.organization.id,
+                    "project_id": self.project.id,
+                    "metric_id": self.tx_user_metric,
+                    "timestamp": user_ts,
+                    "tags": {
+                        self.tx_satisfaction: indexer.record(
+                            self.organization.id, TransactionSatisfactionTagValue.FRUSTRATED.value
+                        ),
+                    },
+                    "type": "s",
+                    "value": [2],
+                    "retention_days": 90,
+                },
+            ],
+            entity="metrics_sets",
+        )
```

Comment:
Two comments:

1. Should we also send a bucket with a different `satisfaction` value here, so we know for sure that we're not accidentally counting all users?
2. To save some lines of code, you could collapse the two `frustrated` buckets into one, with `"value": [1,2]`.